tests/SharedCodeAllocator.test.cpp

```diff
@@ -175,6 +175,8 @@ TEST_CASE("NativeModuleRefRefcounting")
     REQUIRE(modRefA->getRefcount() == 1);
     REQUIRE(modRefB->getRefcount() == 1);
 
+#if defined(__linux__) && defined(__GNUC__)
```

Comment:
I just decided to not generate this test on linux + gnuc because of the annoying self-move warning